index.js

```diff
@@ -57,8 +57,10 @@ function checkDeployment(deployment) {
 }
 
 async function waitForDeployment() {
-  console.log(github.context.payload)
-  const sha = github.context.payload.head_commit.id
+  let sha = core.getInput('commit-id')
+  if (!sha) {
+    sha = github.context.payload.head_commit.id
+  }
```

Comment:
```suggestion
  const sha = core.getInput('commit-id') || github.context.payload.head_commit.id
```

Is a more declarative option. Not really better or worse